src/api.ts

```diff
@@ -11,3 +12,15 @@ export function getMovie(query: string): Promise<MovieData | ResponseError> {
       Error: 'unexpected error',
     }));
 }
+
+export function normalizeMovieData(movieData: MovieData): Movie {
+  return {
+    title: movieData.Title,
+    description: movieData.Plot,
+    imgUrl: movieData.Poster !== 'N/A'
```

Comment:
move this to the separate variable
```suggestion
    imgUrl: ...
```